Detectors/TPC/qc/include/TPCQC/Clusters.h

```diff
@@ -49,9 +49,9 @@ class Clusters
 
   void fillADCValue(int cru, int rowInSector, int padInRow, int timeBin, float adcValue);
 
-  void normalize();
+  void normalize(const float nHBFPerTF = 128);
```

Comment:
There is a data member for the number of HBFs. So it is not needed twice
```suggestion
  void normalize();
```